crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -41,17 +30,16 @@ async fn main() -> Result<()> {
 }
 
 async fn run() -> Result<()> {
-    let args = Cli::parse();
+    let conf = match config::get() {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            eprintln!("failed to read configuration: {e}");
+            std::process::exit(2);
```

Comment:
why 2?

Reply:
I've always coded failing to read essential config for CLI tools with exit status 2, but open to suggestions

Reply:
Changing to 78 after reading  https://freedesktop.org/software/systemd/man/systemd.exec.html#Process%20Exit%20Codes